conjur/init/init_controller.py

```diff
@@ -45,24 +45,43 @@ def load(self):
         self.write_conjurrc()
 
         sys.stdout.write("Configuration initialized successfully!\n")
-        sys.stdout.write("To begin using the CLI, log in to the Conjur server by" \
-                         "running `conjur authn login`")
+        sys.stdout.write("To begin using the CLI, log in to the Conjur server by " \
+                         "running `conjur login`\n")
 
     def get_server_certificate(self):
         """
         Method to get the certificate from the Conjur endpoint detailed by the user
         """
         # pylint: disable=line-too-long
         if self.conjurrc_data.appliance_url is None:
-            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ")
+            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ").strip()
+            if self.conjurrc_data.appliance_url is None or self.conjurrc_data.appliance_url == '':
```

Comment:
It cannot be None here, it would throw when you call `strip()` in the previous line.
So You can remove it:
```suggestion
            if self.conjurrc_data.appliance_url == '':
```